src/Tribe/Rewrite.php

```diff
@@ -750,6 +760,8 @@ protected function get_dynamic_matchers( array $query_vars ) {
 	 * Returns a list of post types supported by the implementation.
 	 *
 	 * @since 4.9.11
+	 *
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: da8dc7395f707777790fdcd23887bbaa</sub>